src/System.Net.WebSockets.Client/src/System/Net/WebSockets/WinRTWebSocket.cs

```diff
@@ -340,7 +340,6 @@ public override void Dispose()
                     _webSocketReceiveResultTcs = new TaskCompletionSource<WebSocketReceiveResult>();
                 }
 
-                InterlockedCheckValidStates(s_validAfterReceiveStates);
```

Comment:
Do we have anything similar in the other implementations that we need to fix similarly?  Or maybe this was already removed from those (if it was ever there)?